### ISSUE TITLE
Related field in Select component is not showing on a form for creating a new record

### ISSUE THREAD

**Issue by DriveSoft** [premieroctet/next-admin#153]
A bug after updating to version 3.2.6, 
A related field in Select component is not showing on a form for creating a new record.

I have on form Select component to pick a city which shows: City (Country)

```
      City: {
         title: "Cities",
         toString: (city) => `${city.name} (${city.country?.name})`,
      },
```

But instead to see a country name in parenthesis, I see "undefined"
![image](https://github.com/premieroctet/next-admin/assets/11293522/93bf687c-f3b4-4c33-a4d0-942c3defc3fd)

But it happens only on form for creating record, when I'm editing a record, then the country name is showing in a Select component.

**Comment by cregourd** [comment]
Hi @DriveSoft,

We've reproduce this case, we will work on it soon

**Comment by cregourd** [comment]
This bug was fix on [v3.2.7](https://github.com/premieroctet/next-admin/blob/%40premieroctet/next-admin%403.2.7/packages/next-admin/CHANGELOG.md#327) 🚀

Feel free to give feedback
